velox/dwio/parquet/tests/writer/SinkTests.cpp

```diff
@@ -84,4 +70,4 @@ TEST_F(SinkTest, abort) {
   // Abort would not flush.
   writer->abort();
   ASSERT_EQ(size, fs::file_size(filePath));
-}
+}
```

Comment:
empty line

Reply:
Did you mean add an empty line? format-check is failing because of the extra empty line.

Reply:
I mean add an empty line at the end of the file.

Reply:
yes , I did that, format-check throws the empty line as an error.

Reply:
It's weird. Let's run `python scripts/check.py format commit --fix` and let the checker fix it automatically.

Reply:
PS: This should be run after committing all your changes locally.